### ISSUE TITLE
Similarity score NaN when comparing two exact similar jsons containing empty array

### ISSUE THREAD

**Issue by aymar99** [deblockt/json-diff#12]
Steps to reproduce :

- Use 0.0.8 version of the library.

- Use 

```        final var jsonMatcher = new CompositeJsonMatcher(
new LenientJsonArrayPartialMatcher(), // comparing array using lenient mode (ignore array order and extra items)
new LenientJsonObjectPartialMatcher(), // comparing object using lenient mode (ignoring extra properties)
new LenientNumberPrimitivePartialMatcher(new StrictPrimitivePartialMatcher()) // comparing primitive types and manage numbers (100.00 == 100)
);
```

Pass the below json for both expected and actual,

`{
"a":[]
}
`

Print the similarity score, expected 100, actual NaN

**Comment by aymar99** [comment]
I feel this method is where the fix is required, https://github.com/deblockt/json-diff/blob/9dfab1e96c2d60228314f76cd91ad5c6534728a2/src/main/java/com/deblock/jsondiff/diff/JsonArrayDiff.java#L39
When the array size is zero, here there could be 0/0 error.

**Comment by deblockt** [comment]
Hi, 

Thanks for this feedback.
It's fixed in 0.0.9 version.
